docs/storage_controller.md

```diff
@@ -0,0 +1,147 @@
+# Storage Controller
+
+## Concepts
+
+The storage controller sits between the user and the pageserver, and handles the details of mapping tenants to pageserver tenant shards.
+
+It implements a pageserver-compatible API that may be used for CRUD operations on tenants and timelines, translating these requests into appropriate operations on the shards within a tenant, which may be on many different pageservers. Using this API, the storage controller may be used as "virtual pageserver" that
```

Comment:
nit: I'm not a big fan on the "virtual pageserver" metaphor since one can't use the storage controller to server get page requests. I feel like this metaphor is more useful for the cp team.

Reply:
Fair.  reworded to specifically say admin API.